editorial_material/editorial_policies.tex

```diff
@@ -0,0 +1,84 @@
+\documentclass{article}
+\usepackage{graphicx}
+\usepackage{hyperref}
+
+\begin{document}
+
+\title{Editorial Policies of LiveCoMS}
+\author{Editorial Board of LiveCoMS}
+
+\maketitle
+
+\begin{abstract}
+The current draft list of editorial board policies for the Living
+Journal of Computational Molecular Science (LiveCoMS). These policies
+are not yet active, as they are still in draft.
+\end{abstract}
+
+\section{Policies on Submissions to LiveCoMS}
+
+\subsection{Policy on Licensing}
+
+LiveCoMS does not ask for or request any copyright transfer.
+
+However, in order to submit to LiveCoMS, authors must provide, at
+minimum, a license for LiveCoMS to publish the article and distributed
+it free of charge. We recommend that the authors release the article
+under an open source license such as \href{Creative Commons
+  Attribution}{https://creativecommons.org/licenses/by/4.0/},
+releasing the document for anyone to copy and redistribute the
+material in any medium or format, and remix, transform, and build upon
+the material for any purpose, even commercially.  Making it available
+to all of course makes it possible for LiveCoMS to publish it, and for
+the community to edit and contribute.
+
+Other more restrictive licenses may be possible as long as LiveCoMS
+has the permission to publish and excerpt from the document; this
+might be required if someone other than the authors has some rights to
+the material (for example, if it was previously published in some
+other journal). Generally, we only allow a more restrictive license in
+these cases, but are happy to discuss. Please ask the managing editors
+if you require some other licensing regime.
+
+\subsection{Policies on Author Order}
+Authors should, to the extent possible, determine the author order among themselves. 
+
+\subsection{Policy on Abandoned Documents}
+
+All submitters agree by submission of the document to so that if
```

Comment:
I would rephrase this paragraph entirely as it is too vague and does not actually specify how they have given consent. I think we want them to indicate they are explicitly agreeing. Perhaps something more like this:

"By the submission of their paper, document, or materials, all the authors consent that if they no longer are willing or able to maintain their work, LiveCoMS may assign another individual or individuals to do so, with appropriate modifications to the authors list. Authors would be given written notice (by e-mail and formal letter) if this were to happen and would have a period of six months to respond and/or designate a successor before LiveCoMS would do so for them. Typically, authors are *implicitly* giving a right to do this via licensing under \href{Creative Commons
 +  Attribution}{https://creativecommons.org/licenses/by/4.0/} or similar licenses which give others the right to create derivative works (potentially allowing others to ``resurrect'' a document which has been abandoned); however, we expect all authors to *explicitly* consent to this policy to avoid any confusion. Ordinarily, we expect this policy will be relevant only in unusual or extreme cases where an author or authors dies or leaves the field; in most other cases authors will presumably be available to designate their own successors or succession plan if a work is valuable to the field and will continue to need maintenance and the original author(s) are no longer willing or able to do so. However, we want to plan for the possibility of unusual events, hence our need for this policy."

Reply:
Done.